files/zh-cn/learn/performance/multimedia/index.md

```diff
@@ -0,0 +1,153 @@
+
+<table>
+  <tbody>
+    <tr>
+      <th scope="row">前提：</th>
+      <td>
+        基本的计算机操作能力，
+        <a
+          href="/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software"
+          >已安装基本软件</a
+        >，以及基本的
+        <a href="/zh-CN/docs/Learn/Getting_started_with_the_web"
+          >客户端 Web 技术知识</a
+        >。
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">目标：</th>
+      <td>
+        了解各种图像格式、它们对性能的影响，以及如何减少图像对页面整体加载时间的影响。
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+> **备注：** 这是对 Web 上多媒体传输进行优化的概要介绍，只涵盖了一般原理和技术。如需更深入的指南，请参阅 <https://images.guide>。
+
+## 为何优化多媒体？
+
+对于一般网站而言，[其带宽的 51% 来自图像，其次是视频占 25%](https://discuss.httparchive.org/t/state-of-the-web-top-image-optimization-strategies/1367)，因此可以说处理和优化多媒体内容非常重要。
+
+你需要考虑数据使用情况。许多人使用的是有限流量套餐，甚至是按使用量付费，每兆字节都要付费。这不仅仅是新兴市场的问题。截至 2018 年，[英国仍有 24% 的用户使用按使用量付费](https://www.ofcom.org.uk/__data/assets/pdf_file/0021/113169/Technology-Tracker-H1-2018-data-tables.pdf)。
+
+你还需要考虑内存问题，因为许多移动设备的 RAM 有限。请记住，当图像下载后，它们需要存储在内存中。
+
+## 优化图像传递
+
+尽管图像下载是带宽消耗最大的部分，但其对[感知性能](/zh-CN/docs/Learn/Performance/Perceived_performance)的影响远低于许多人的预期（主要是因为页面文本内容可以立即下载，用户可以在图像到达时看到它们被渲染）。然而，为了提供良好的用户体验，访问者仍然应尽快看到图像。
+
+### 加载策略
+
+对于大多数网站来说，最大的改进之一是[延迟加载](/zh-CN/docs/Web/Performance/Lazy_loading)可视区域以下的图像，而不是无论访问者是否滚动页面查看都在初始页面加载时全部下载它们。许多 JavaScript 库可以为你实现此功能，例如 [lazysizes](https://github.com/aFarkas/lazysizes)，浏览器供应商目前正在开发一种原生的 `lazyload` 属性，目前处于实验阶段。
+
+除了加载一部分图像之外，接下来你应该考虑图像本身的格式：
+
+- 你是否加载了最优的文件格式？
+- 你是否对图像进行了良好的压缩？
+- 你是否加载了正确的尺寸？
+
+#### 最优格式
+
+最优的文件格式通常取决于图像的特点。
+
+> **备注：** 有关图像类型的一般信息，请参阅[图像文件类型和格式指南](/zh-CN/docs/Web/Media/Formats/Image_types)。
+
+[SVG](/zh-CN/docs/Web/Media/Formats/Image_types#svg_可缩放矢量图形) 格式更适合颜色较少且不太逼真的照片的图像。这需要源文件以矢量图形格式提供。如果这样的图像仅以位图形式存在，则应选择 [PNG](/zh-CN/docs/Web/Media/Formats/Image_types#png_便携式网络图形) 作为后备格式。这种类型的图案示例是徽标、插图、图表或图标（注意：SVG 比图标字体好得多！）。两种格式都支持透明度。
+
+PNG 可以以三种不同的输出组合进行保存：
+
+- 24 位色 + 8 位透明度——提供全色准确性和平滑透明度，以牺牲大小为代价。你可能希望为了 WebP（见下文）而避免此组合。
+- 8 位色 + 8 位透明度——提供不超过 255 种颜色，但保持平滑的透明度。大小不太大。这些可能是你想要的 PNG。
+- 8 位色 + 1 位透明度——提供不超过 255 种颜色，并仅为每个像素提供无或完全透明度，这使透明边界看起来生硬并且有锯齿。尺寸小，但视觉保真度付出了代价。
+
+优化 SVG 的一个不错的在线工具是 [SVGOMG](https://jakearchibald.github.io/svgomg/)。对于 PNG，有 [ImageOptim 在线](https://imageoptim.com/online)或 [Squoosh](https://squoosh.app/)。
+
+对于不具有透明度的摄影素材，有更广泛的格式可供选择。如果要谨慎行事，那么你应该选择压缩良好的**渐进式 JPEG**。渐进式 JPEG 与普通 JPEG 不同，它们是渐进式显示的（因此得名），这意味着用户先看到的是一个低分辨率的图像版本，随着图像下载而逐渐清晰，而不是图像以全分辨率从上到下加载，或者只有在完全下载后才显示。对于这些图片，一个好的压缩工具是 MozJPEG，例如能够在在线图像优化工具 [Squoosh](https://squoosh.app/) 中使用。75% 的质量设置能够得到优秀的结果。
+
+其他格式在压缩方面改进了 JPEG，但并非所有浏览器都支持：
+
+- [WebP](/zh-CN/docs/Web/Media/Formats/Image_types#webp_图像)——既适用于图像又适用于动图的绝佳选择。WebP 提供比 PNG 或 JPEG 更好的压缩，支持更高的色深、动画帧和透明度等（但不支持渐进式显示）。除 macOS 桌面版 Safari 14 外，所有主流浏览器都支持它。
+
+  > **备注：** 尽管[宣布](https://developer.apple.com/videos/play/wwdc2020/10663/?time=1174) 在 Safari 14 中支持 WebP，但在版本 14.0 中，.webp 图像在 macOS 桌面上不显示原生图像，而 Safari 在 iOS 14 上可以正确显示 .webp 图像。
```

Comment:
Need to confirm whether webp images can display properly on macOS desktop 14.0+, it determines whether 'as of version 14.0' should be translated into '截至 14.0 版本为止' or '自从 14.0 版本开始'.

ping @yin1999 because he seems to be a macOS user 👀 

```suggestion
  > **备注：** 尽管 Safari 14 已[宣布支持](https://developer.apple.com/videos/play/wwdc2020/10663/?time=1174) WebP，但截至 14.0 版本为止，.webp 图像无法在 macOS 桌面上原生显示，而 iOS 14 上的 Safari 却能正常显示 .webp 图像。
```

Reply:
After looking up [caniuse](https://caniuse.com/webp), I feel like we need more precise description here.

Reply:
raised https://github.com/mdn/content/pull/29353